contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -40,11 +40,14 @@ contract TrueCreditAgency is UpgradeableClaimable {
     mapping(ITrueFiPool2 => mapping(address => uint8)) public creditScore;
     mapping(ITrueFiPool2 => mapping(address => uint256)) public borrowed;
     mapping(ITrueFiPool2 => mapping(address => uint256)) public totalPaidInterest;
+    mapping(ITrueFiPool2 => mapping(address => uint256)) public nextInterestRepayTime;
 
     mapping(ITrueFiPool2 => bool) public isPoolAllowed;
 
     mapping(address => bool) public isBorrowerAllowed;
 
+    uint256 public repaymentPeriod;
```

Comment:
Let's update this name with the result of this discussion thread: https://github.com/trusttoken/smart-contracts/pull/780#discussion_r675704796